src/v/cluster/members_manager.cc

```diff
@@ -178,9 +178,16 @@ ss::future<> members_manager::handle_raft0_cfg_update(
               [cfg = std::move(cfg), update_offset](members_table& m) mutable {
                   m.update_brokers(update_offset, cfg.brokers());
               })
-            .then([this, diff = std::move(diff)]() mutable {
+            .then([this, diff = std::move(diff), update_offset]() mutable {
```

Comment:
This seems reasonable, but it makes me wonder what else breaks if we're seeing these updates out of order.  Surely update_allocation_nodes above would also have a problem?  (I'm missing something about how the code works, not claiming the change is wrong)

Reply:
@mmaslankaprv does this question raise any concerns about other places that consume the updateS?